macros/multiple_databases/timestamp_from_date.sql

```diff
@@ -32,6 +32,13 @@
                 3
             )
     end
+{%- elif target.type == 'databricks' -%}
+    case
+        when length({{ date_field }}) > 0
+            then to_timestamp({{ date_field }}, 'yyyy-MM-dd')
+        else
+            null
+    end
```

Comment:
- Why do we check for `length() > 0`? Can we instead use `try_to_timestamp`?
- Why do you provide the format? Does it also work without?
- When we still need the `null` else clause, make it type casted.